server/services/offlinePayment.service.js

```diff
@@ -160,7 +166,11 @@ export const resolveOfflinePayment = async ({ offlinePaymentId, adminId }) => {
       throw new ErrorHandler(httpStatus.INTERNAL_SERVER_ERROR, 'Internal Server Error', error);
     });
 
-    await createNotification(NOTIFICATIONS.OFFLINE_PAYMENT_RESOLVED, offlinePayment.userId);
+    const description = `Your payment of N${offlinePayment.amount} has been confirmed`;
```

Comment:
Can you create a method for getting/displaying the price which returns the formatted price (in commas, with the currency)? This is to ensure that we have the same formatting for price all through the application.

Something like:

```suggestion
    const description = `Your payment of ${getMoneyFormat(offlinePayment.amount)} has been confirmed`;
```